Bindings/bindings.json

```diff
@@ -1359,6 +1359,57 @@
                     "help": "Choose platform to send native notifcations. Select Template to send template notification."
                 }
             ]
+        },
+        {
+            "type": "sendGrid",
+            "displayName": "SendGrid",
+            "direction": "out",
+            "settings": [
+                {
+                    "name": "to",
+                    "value": "string",
+                    "defaultValue": "",
+                    "required": false,
+                    "label": "To address",
+                    "help": "To address that the message should be sent to.",
+                    "validators": [
+                        {
+                            "expression": "TODO",
```

Comment:
got it from the portal.azure.com aad portal extension
^(|[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4})$